extensions/amp-analytics/0.1/variables.js

```diff
@@ -234,20 +234,26 @@ export class VariableService {
     this.register_('SCROLL_LEFT', () =>
       Services.viewportForDoc(this.ampdoc_).getScrollLeft()
     );
-    // Was set async before
-    this.register_('FIRST_CONTENTFUL_PAINT', () => {
-      return Services.performanceFor(
-        this.ampdoc_.win
-      ).getFirstContentfulPaint();
-    });
-
-    this.register_('FIRST_VIEWPORT_READY', () => {
-      return Services.performanceFor(this.ampdoc_.win).getFirstViewportReady();
-    });
 
-    this.register_('MAKE_BODY_VISIBLE', () => {
-      return Services.performanceFor(this.ampdoc_.win).getMakeBodyVisible();
-    });
+    // Call performance-impl.js to get promise returning value for metric
+    this.register_('FIRST_CONTENTFUL_PAINT', () =>
```

Comment:
These variables should not be exposed to analytics component within FIE. 
Does FIE has its own instance of ampdoc and performance metrics?

Reply:
I'm not familiar with the FIE use case. Can you share a little more context? Thanks!

Reply:
@jridgewell  Do you know if FIE will have its own instance of ampdoc services? 
If not, you'll need to apply the check `isInFIE()`, and return different values based on the result. 
cc @powerivq to comment on the FIE use case.

Reply:
https://github.com/ampproject/amphtml/blob/34c3f78535608d7cdd3061756088308561891764/extensions/amp-analytics/0.1/variables.js#L258-L262

`COOKIE` is an example

Reply:
FIE shares (for now) with the main ampdoc. Do you think FIE shouldn't receive these metrics? I'm not sure that an analytics inside an FIE really wants the metrics for the FIE, I think they'd want the main ampdoc.

Reply:
Synced with @kevinkimball offline. We decided not to share the performance metrics to analytics within the FIE. (It's probably ok, but without privacy review we decided to prevent ads from accessing the parent docs performance metrics by default). 
Given that FIE shares the same ampdoc and the same performanceService with the parent doc, we need to apply the `isInFie()` check before resolving the macros.

Reply:
Yes, I am working on this now and will hide from FIE